client/devdocs/design/search-collection.jsx

```diff
@@ -93,9 +88,7 @@ const Collection = ( {
 						component={ component }
 						section={ section }
 					/>
-					{ component && (
-						<ReadmeViewer readmeFilePath={ readmeFilePath } showEditLink={ showEditLink } />
```

Comment:
Note that the `showEditLink` prop has been added in the same PR that added the whole Gutenberg components in the DevDocs (#26314), and is not used anywhere else.
I guess we could remove it from `ReadmeViewer` (and its tests) as well.

Reply:
> I guess we could remove it from ReadmeViewer (and its tests) as well.

Hmm, I still see a couple its usages that are not related to Gutenberg components and blocks: 🤔 
https://github.com/Automattic/wp-calypso/search?p=1&q=%22ReadmeViewer%22&unscoped_q=%22ReadmeViewer%22

Reply:
Sorry, I meant remove the `showEditLink` prop from `ReadmeViewer` (and its tests), not the entire component 🙂

Reply:
Oh sorry, it seems I wasn't fully awake while reading your comment. 🤦‍♂️ That's exactly what you wrote the first time around, but I somehow managed to completely misunderstand it. 😄 
Anyway, I'll clean it up in a follow-up.